src/frame.js

```diff
@@ -0,0 +1,14 @@
+//each frame initializes with empty array
+
+function Frame(){
+  this.outcome = []
+}
+
+Frame.prototype.start = function(){
+  bowl = new Bowl()
```

Comment:
Should the frame do the bowling (by calling bowl.roll, or should a roll be created with the rolls and then passed into a new frame?